pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -1349,13 +1361,25 @@ type NodeGroupBase struct {
 	// This is a hack, will be removed shortly. When this is true for Ubuntu and
 	// AL2 images a legacy bootstrapper will be used.
 	CustomAMI bool `json:"-"`
+
+	// ContainerRuntime defines the runtime to use for the container.
```

Comment:
just the one container? 🙃 

```suggestion
	// ContainerRuntime defines the runtime (CRI) to use for containers on the node
```

Reply:
Fair point